components/fieldgroups/profile/profile-healthcare-member.schema.json

```diff
@@ -0,0 +1,367 @@
+{
+  "meta:license": [
+    "Copyright 2022 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/profile-healthcare-member",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Healthcare Member Details",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:tags": {
+    "industry": ["healthcare_services"]
+  },
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
+  "description": "Details of a person/member/subscriber who may be waiting for a care or may be receiving care or may have already received care.",
```

Comment:
Healthcare member details of a person that has or will receive a service or care, such as contact information, primary care physician, and plan information.